torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2221,6 +2221,29 @@ def sample_inputs_max_min_binary(op_info, device, dtype, requires_grad, **kwargs
                   for input_tensor, other_tensor in args_for_binary_op)
     return inputs
 
+def sample_inputs_conv_transpose2d(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    # Ordered as shapes for: input, weight, bias, stride, padding, output_padding, groups
+    cases = (((1, 3, 4, 4), (3, 3, 3, 3), (3), (2, 2), 2, (1, 1), 1),
+             ((2, 2, 4, 4), (2, 2, 4, 5), (4), (3, 3), 1, (2, 2), 2),
+             ((1, 1, 4, 5), (1, 1, 4, 3), (1), 2, 1, 1, 1),
```

Comment:
Thanks for the suggestion, @zou3519. I used a single number since it represents a shape, and is passed to `make_arg` which works for integers or tuples as well.

Reply:
Update: having an int there raises mypy errors (the combined type becomes `builtins.object` which is not iterable). Having a tuple there seems like a better solution `(1,) instead of 1`. Fixed in the recent commit. Thanks!